src/store/student-store.js

```diff
@@ -8,6 +9,16 @@ students[123456] = {
     name: 'John von Neumann',
     items: []
 };
+
+students[111111] = {
+    id: 111111,
+    name: 'Boaty McBoatface',
+    items: [{
+        address:'4324234',
+        timestamp: 0
```

Comment:
For my usage of this, we need to use an actual item Address and add it's corresponding model Address.  m8y7nFLsT and  iGwEZVeaT work. 

 (note that despite using it this way, the app will not consider the item to be checked out, not sure if this is a negative, but it does let us still use the item for testing checkout in other places)